### DISCUSSION TITLE
Setup not running

### DISCUSSION THREAD

**Discussion by Chipmunk1999** [joncage/ed-scout#202] [Q&A]
Hello,

I've tried running the setup for 1.8.2 on Windows 10 Pro.  The program does not run.  It asks me if I would like to allow the program to make changes to my hard drive, but then nothing happens after I accept.  I've also tried using the diagnostic build and running the exe through the zip file.  Any help would be appreciated.

**Comment by nmgsimulations** [comment]
Same issue here. I have a very old version of ED Scout, not sure which version it is, that works. See attached. It is not the latest, but it works!

[EDScout.zip](https://github.com/joncage/ed-scout/files/7702607/EDScout.zip)
.

**Reply by Trickyhicky** [reply]
Thanks, @nmgsimulations your solution has returned service to normal for me.

**Reply by Trickyhicky** [reply]
This solution has worked for me thanks to @nmgsimulations guys 
 [https://github.com/joncage/ed-scout/discussions/202](url)
